### ISSUE TITLE
format html docment error 

### ISSUE THREAD

**Issue by cways** [Microsoft/vscode#18081]
- VSCode Version: Code 1.8.1 (ee428b0eead68bf0fb99ab5fdc4439be227b6281, 2016-12-19T14:42:49.304Z)
- OS Version: Darwin x64 16.3.0
- Extensions:

|Extension|Author|Version|
|---|---|---|
|vscode-great-icons|emmanuelbeziat|1.1.36|
|vscode-icons|robertohuertasm|4.3.0|

---

Steps to Reproduce:

![qq20170104-142437-hd](https://cloud.githubusercontent.com/assets/6712443/21633669/0fcd9104-d28c-11e6-96a0-e61e8a4283d8.gif)

**Comment by aidesigner** [comment]
I am experiencing the same html formatting issue.  After some experimenting the problem shows itself once a html tag with a "style" attribute is encountered.  Perhaps a hotfix is possible, as the broken formatted html code is extremely difficult to read.

**Comment by LostInBrittany** [comment]
I have the same problem with the style tag: 

Intended indentation:
```
<link rel="import" href="../bower_components/polymer/polymer.html">

<dom-module id="test">

  <template>

    <style>
    </style>

    <h1>Hello</h1>

  </template>

  <script>
    Polymer({
      is:"test"
    });
  </script>

</dom-module>
```

Obtained indentation:

```
<link rel="import" href="../bower_components/polymer/polymer.html">

<dom-module id="test">

  <template>

    <style>
    </style>

<h1>Hello</h1>

</template>

<script>
    Polymer({
      is: "test"
    });
  </script>

</dom-module>
```

And I fully agree with @aidesigner , right now VSC messes every HTML file I open, a hotfix would be really helpful :)

**Comment by LostInBrittany** [comment]
The problem seems corrected in the current *insiders version*, `1.9.0-insider`

**Comment by aeschli** [comment]
duplicate of #17849
